src/OSPSuite.Presentation/Presenters/ObservedData/BaseDataRepositoryDataPresenter.cs

```diff
@@ -37,7 +37,7 @@ protected BaseDataRepositoryDataPresenter(TView view) : base(view)
 
       public bool IsLatched { get; set; }
 
-      protected abstract DataTable MapDataTableFromColumns(IEnumerable<DataColumn> dataColumns);
+      protected abstract DataTable MapDataTableFromColumns();
```

Comment:
This parameters does not belong in the method. I stumbled upon that as I was trying to make sense of the exception